include/emitc/Target/Cpp/CppEmitter.h

```diff
@@ -33,33 +33,32 @@ template <typename ForwardIterator, typename UnaryFunctor,
           typename NullaryFunctor>
 inline LogicalResult
 interleaveWithError(ForwardIterator begin, ForwardIterator end,
-                    UnaryFunctor each_fn, NullaryFunctor between_fn) {
+                    UnaryFunctor eachFn, NullaryFunctor betweenFn) {
```

Comment:
This uses the exact same naming as an upstream function, see [here](https://llvm.org/doxygen/namespacellvm.html#ad75c79282c4734018fe1482b12d03d62).

Reply:
Indeed, but `clang-tidy` complains anyway and resulted in a build failure of https://reviews.llvm.org/D104632.

Reply:
Okay that's a compelling reason.